instance/conf.py

```diff
@@ -9,7 +9,20 @@
 # MAIN FLASK APPLICATION
 DEBUG = os.environ.get('FLASK_DEBUG', False)
 SECRET_KEY = os.environ.get('SESSION_SECRET', str(uuid.uuid4()))
-SQLALCHEMY_DATABASE_URI = make_database_url()
+DB_USER = os.environ.get('DB_USER', 'postgres')
+DB_PASSWORD = os.environ.get('DB_PASSWORD', '')
+DB_HOST = os.environ.get('DB_HOST', '127.0.0.1')
+DB_PORT = os.environ.get('DB_PORT', '5432')
+DB_NAME = os.environ.get('DB_NAME', 'qualitas_db')
+DB_URL = os.environ.get('DB_URL')
+SQLALCHEMY_DATABASE_URI = make_database_url(
+    DB_USER=DB_USER,
+    DB_PASSWORD=DB_PASSWORD,
+    DB_HOST=DB_HOST,
+    DB_PORT=DB_PORT,
+    DB_NAME=DB_NAME,
+    DB_URL=DB_URL,
```

Comment:
What is DB_URL being used for here? I don't see it being used in the `make_database_url` function.